media-libs/webrtc-audio-processing/files/webrtc-audio-processing-0.3-Add-generic-byte-order-and-pointer-size-detection.patch

```diff
@@ -0,0 +1,33 @@
+From: Than <than@redhat.com>
```

Comment:
Please always add references to the top of patches, including relevant bugs (Gentoo or otherwise) & the patch source.